docs/indepth/installation.rst

```diff
@@ -270,14 +270,16 @@ The following commands will install the two repositories and the ``clusterhq-flo
 Paste them into a root console on the target node:
 
 .. task:: install_flocker fedora-20
-   :prompt: [root@fedora]#
-
+   :prompt: [root@node]#
+   
 Installing ``flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
 To enable and start Docker, run the following commands in a root console:
 
 .. task:: enable_docker fedora-20
    :prompt: [root@fedora]#
 
+Finally, you will need to run the ``flocker-ca`` tool that is installed as part of the CLI package.
```

Comment:
Or hm, I guess you do need to do it before enabling services... OK, maybe just leave as is then.